controllers/storageautoscaler/storageautoscaler_controller_test.go

```diff
@@ -0,0 +1,27 @@
+package storageautoscaler
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCheckIfScalingNotRequired(t *testing.T) {
+	usage := 5.5
+	threshold := 6
+
+	notRequired := checkIfScalingNotRequired(usage, threshold)
```

Comment:
Checking if scaling is "required" is easier to understand than if it is "not required". Then you have to understand double negatives if it returns false, that it is "not not required". :) For maintainability, I would recommend changing it
```suggestion
	notRequired := checkIfScalingRequired(usage, threshold)
```